js/main.js

```diff
@@ -13,17 +13,23 @@
   function addTask() {
     const content = textAdd.value;
     task.content = content
-    todos.push(task)
+    todos.push({
+      content: content,
+      status: '作業中'
+    })
     textAdd.value = ''
   }
-  function displyTask(todos) {
-    const tr =  document.createElement('tr');
-    const idTd = document.createElement('td');
-    const td = document.createElement('td');
-    todos.forEach(task => {
+
+  function displayTask(todos) {
```

Comment:
関数は下記のような構文もありますので覚えておきましょう。
```suggestion
  const displayTask = function (todos) {
```

Reply:
`const displayTask = (todos) => {`